cfme/tests/test_rest.py

```diff
@@ -179,244 +174,314 @@ def test_user_settings(rest_api):
 
-@pytest.mark.uncollectif(lambda: current_version() < '5.7')
-def test_bulk_query_groups(rest_api):
-    """Tests bulk query on 'groups' collection
+        Metadata:
+            test_flag: rest
+        """
+        collection = rest_api.collections.groups
+        data0, data1 = collection[0]._data, collection[1]._data
+        response = rest_api.collections.groups.action.query(
+            {'description': data0['description']}, {'description': data1['description']})
+        assert rest_api.response.status_code == 200
+        assert len(response) == 2
+        assert data0 == response[0]._data and data1 == response[1]._data
 
-    Metadata:
-        test_flag: rest
-    """
-    collection = rest_api.collections.groups
-    data0, data1 = collection[0]._data, collection[1]._data
-    response = rest_api.collections.groups.action.query(
-        {"description": data0["description"]}, {"description": data1["description"]})
-    assert len(response) == 2
-    assert data0 == response[0]._data and data1 == response[1]._data
 
+class TestArbitrationSettingsRESTAPI(object):
+    @pytest.mark.uncollectif(lambda: current_version() < '5.7')
+    def test_create_arbitration_settings(self, request, rest_api):
+        """Tests create arbitration settings.
 
-@pytest.mark.uncollectif(lambda: current_version() < '5.7')
-def test_create_arbitration_settings(request, rest_api):
-    """Tests create arbitration settings.
+        Metadata:
+            test_flag: rest
+        """
+        num_settings = 2
+        response = arbitration_settings(request, rest_api, num=num_settings)
+        assert rest_api.response.status_code == 200
+        assert len(response) == num_settings
+        record = rest_api.collections.arbitration_settings.get(id=response[0].id)
+        assert record._data == response[0]._data
 
-    Metadata:
-        test_flag: rest
-    """
-    num_settings = 2
-    response = arbitration_settings(request, rest_api, num=num_settings)
-    assert len(response) == num_settings
-    collection = rest_api.collections.arbitration_settings
-    record = collection.get(id=response[0].id)
-    assert record._data == response[0]._data
+    @pytest.mark.uncollectif(lambda: current_version() < '5.7')
+    @pytest.mark.parametrize('method', ['post', 'delete'])
+    def test_delete_arbitration_settings_from_detail(self, request, rest_api, method):
+        """Tests delete arbitration settings from detail.
 
+        Metadata:
+            test_flag: rest
+        """
+        num_settings = 2
+        response = arbitration_settings(request, rest_api, num=num_settings)
+        assert len(response) == num_settings
+        status = 204 if method == 'delete' else 200
+        for entity in response:
+            entity.action.delete(force_method=method)
+            assert rest_api.response.status_code == status
+            with error.expected('ActiveRecord::RecordNotFound'):
+                entity.action.delete()
```

Comment:
**OPTIONAL**

Consider expanding this test coverage a bit - using the parametrized method on the 2nd delete here, or trying to delete a non-existant record with both methods in both parametrized test cases.